saopaulo2024-2-edicao.html

```diff
@@ -0,0 +1,603 @@
+  <header id="header">
+    <div class="container">
+
+      <div id="logo" class="pull-left">
+        <a href="#intro" class="scrollto"><img src="img/logo.png" alt="" title=""></a>
+      </div>
+
+      <nav id="nav-menu-container">
+        <ul class="nav-menu">
+          <li class="menu-active"><a href="#intro">Home</a></li>
+          <li><a href="#about">Sobre</a></li>
+          <li><a href="#schedule">Cronograma</a></li>
+          <li><a href="#faq">FAQ</a></li>
+          <li><a href="#supporters">Patrocinadores</a></li>
+          <li><a href="#venue">Local</a></li>
+          <li><a href="#speakers">Organizadoras</a></li>
+          <li><a href="#gallery">Fotos</a></li>
+        </ul>
+      </nav><!-- #nav-menu-container -->
+    </div>
+  </header><!-- #header -->
+
+  <!--==========================
+    Intro Section
+  ============================-->
+  <section id="intro" class="dark">
+    <div class="intro-container wow fadeIn">
+      <h1 class="mb-4 pb-0">Rails Girls São Paulo 2024<br><p class="mb-4 pb-0">Segunda Edição</p></h1>
+      
+      <p class="mb-4 pb-0">09 de Novembro</p>
+      <p class="nav-tabs">Inscrições Em Breve</a>
+    </div>
+  </section>
+
+  <main id="main">
+
+    <!--==========================
+      About Section
+    ============================-->
+    <section id="about">
+      <div class="container">
+        <div class="row">
+          <div class="col-lg-8">
+            <h2>Sobre o evento</h2>
+            <p>O <strong>Rails Girls</strong> é um projeto sem fins lucrativos, que proporciona a mulheres de todas as idades e que tem
+            <strong>pouca ou nenhuma experiência com programação</strong> uma ótima experiência inicial em desenvolvimento de software.
+            A ideia é mostrar que TI não é um "bicho de sete cabeças" e que <strong>elas podem, SIM, fazer parte desse mundo!</strong></p>
+          </div>
+          <div class="col-lg-2">
+            <h3>Onde</h3>
+            <img src="img/logo-iugu-branco.png" alt="Logo IUGU" class="img-fluid"><br><br>
+            <p>Av. das Nações Unidas, 12495 | Cidade Monções | São Paulo/SP  <p><a href="https://maps.app.goo.gl/z9FmUVaCG9D4Wfd29" target="_blank">Ver no mapa</a></p>
+          </div>
+          <div class="col-lg-2">
+            <h3>Quando</h3>
+            <p>Sábado<br>09 de Novembro de 2024</p>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!--==========================
+      Schedule Section
+    ============================-->
+    <section id="schedule" class="section-with-bg">
+      <div class="container wow fadeInUp">
+        <div class="section-header">
+          <h2>Cronograma</h2>
+          <p>Olha tudo de legal que vai acontecer no evento!</p>
+        </div>
+
+        <ul class="nav nav-tabs" role="tablist">
+          <li class="nav-item">
+            <a class="nav-link active" href="#day-1" role="tab" data-toggle="tab">Pré-Evento: Segunda - 04/11</a>

```

Comment:
Achei que não ornou o título...
talvez `Pré-evento` pudesse ser um paragrafo ou nem precise 🤔 
```suggestion
            <a class="nav-link active" href="#day-1" role="tab" data-toggle="tab">Segunda - 04/11</a>

```